department-of-veterans-affairs/caseflow
# Khalin/APPEALS-29419

Resolves [Intake Form Step 2.3: Review Tasks & Appeals - Tasks related to an existing Appeal](https://jira.devops.va.gov/browse/APPEALS-29419)

# Description

- Created "Yes" and "No" radio buttons
- Radio button defaults to "No"
- Conditionally renders related appeal upon selection of "Yes" radio button

## Acceptance Criteria

## Testing Plan

1. First run the following command to ensure the correspondence.rb file has seeded the relevant data for testing - `bundle exec rake db:seed:correspondence`
2. In rails console pull the uuid for the .last created correspondence  
3. Open browser and got to http://localhost:3000/queue/correspondence/:correspondence_uuid/intake   use the UUID found in the rails console for previous  step in the link
4. Click "Continue" to go to step 2
5. Click the "Yes" Radio button and the page should render the related appeal section as referenced in the AC

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ![Screenshot 2023-11-14 at 3 55 30 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/134650238/d0d9a3e7-e413-4383-80fc-4205d0859c93)
|
![Screenshot 2023-11-14 at 3 56 08 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/134650238/9ee056f0-b061-4b97-8d08-cf29f772faf5)


## Storybook Story
*For Frontend (Presentation) Components*
